apps/preview/next/pages/examples/SfListItem.tsx

```diff
@@ -115,7 +115,9 @@ function Example() {
     <ComponentExample controls={{ state, controls }}>
       <SfListItem
         className="max-w-sm"
-        {...state.get}
+        label={state.get.label}
+        as={state.get.as}
+        truncate={state.get.truncate}
```

Comment:
Is that needed?

Reply:
yes its written in description of PR, this led to warning/error (I dont remember) in nextjs, because there is no prop `secondaryText` on this component, as we passed whole state